src/components/app/MenuButtons/index.js

```diff
@@ -18,10 +18,13 @@ import {
 } from 'firefox-profiler/selectors/profile';
 import { getDataSource } from 'firefox-profiler/selectors/url-state';
 import { getIsNewlyPublished } from 'firefox-profiler/selectors/app';
-import { MenuButtonsMetaInfo } from 'firefox-profiler/components/app/MenuButtons/MetaInfo';
+
+/* Note: the order of import is important, from most general to most specific,
+ * so that the CSS rules are in the correct order. */
+import { ButtonWithPanel } from 'firefox-profiler/components/shared/ButtonWithPanel';
```

Comment:
This change is made necessary because I removed the same import from MetaInfo.js =>  the CSS file for ButtonWithPanel was loaded _after_ the one for MetaInfo, and was overriding some properties.